docs/api/ethereum.md

```diff
@@ -742,6 +744,7 @@ A `class` providing the communication with the Ethereum blockchain through [Meta
 | options.gatewayConfig.multiOrderId | A `string` representing an Ethereum address of the [multi order gateway](/#public-addresses).
```

Comment:
| options.gatewayConfig.multiOrderId | A `string` representing an Ethereum address of the [multi-order gateway](/#public-addresses).